style.css

```diff
@@ -11,6 +11,10 @@ li {
   list-style: none;
 }
 
+a {
+  text-decoration: none;
+}
+
```

Comment:
- [ ] Great job Implementing the changes requested, however I think you forgot about the color as pointed out by the previous reviewer https://github.com/HenaryKhuraijam/portfolio/pull/1#discussion_r1070256867 Please fix the logo color to be `color: #6070FF;` this will match the fgma design :+1:

Reply:
Thanks for pointing that out.